tex/chapters/gausss-law/infinite-plane.tex

```diff
@@ -0,0 +1,31 @@
+\section{Infinite plane}
+An infinite plane with charge density per unit of area $\sigma$ in empty space, located on the $XY$-plane is given. Due to the symmetry of a
```

Comment:
Due to the symmetry of THE system, not A system

Reply:
done